README.md

```diff
@@ -172,6 +172,8 @@ Show cache created by EnvyControl
 sudo envycontrol --cache-query
 ```
 
+>[!NOTE]
```

Comment:
Can we remove this special callout now? This is not done anywhere else in the README.

@bayasdev can you weigh in here please?

Reply:
Sorry, we can remove the [!NOTE] to stick with the document style, however I would also make the note as a standard paragraph after the blank line in line 131.